modules/video/src/screenspacevideo.cpp

```diff
@@ -38,13 +38,10 @@
 namespace openspace {
 
 documentation::Documentation ScreenSpaceVideo::Documentation() {
-    documentation::Documentation doc = ScreenSpaceRenderable::Documentation();
+    documentation::Documentation doc = VideoPlayer::Documentation();
```

Comment:
Are you sure this is correct?  The ScreenSpaceVideo needs the documentation for both the screenspacerenderable and the videoplayer component.  This removes the ones from the screenspace renderable, right?

Reply:
The base class members are shown in the documentation so having them here as well means it is duplicated

Reply:
Ah. Yea!